model/src/w3updtmd.F90

```diff
@@ -3004,6 +3130,18 @@ SUBROUTINE W3UTRN ( TRNX, TRNY )
 !/
       END SUBROUTINE W3UTRN
 !/ ------------------------------------------------------------------- /
+!
+!> @author W. E. Rogers, (NRL) @date 06-Dec-2010
+!> @param[in] ZZ Field to calculate derivatives of.
+!> @param[in] ZUNIT Units of ZZ (used for test output).
+!> @param[out] DZZDX Derivative in X-direction (W-E).
+!> @param[out] DZZDY Derivative in Y-direction (S-N).
+!> @returns DZZDX, DZZDY
```

Comment:
Don't use the return tag to describe the [out] parameters. A subroutine does not return anything. You have already correctly documented the two parameters as [out] and that's all you need to do.

So just remove the return tag from all subroutine documentation blocks.

Reply:
That's a good point.  I'll make that change.

Reply:
I'll leave this (and others) unresolved momentarily, then will resolve them all at once when I've re-ran doxygen and reposted the output.